portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,12 +40,27 @@ private String convertListToJson(List<Comment> comments) {
     return json;
   }
 
+  /**
+   * Return the value of the limit parameter if valid, 
+   * otherwise the default value which is 5
+   */
+  private int getLimit(HttpServletRequest request) {
+    int limit;
+    try {
+      limit = Integer.parseInt(getParameter(request, "limit", ""));
```

Comment:
If you want you could create a `getIntParameter` method that returns an `int` and the default value (which would also be an int) if it is not present or fails to parse.

Reply:
It makes sense.